apps/sms/lib/bridge/plugins/dispose/client.js

```diff
@@ -0,0 +1,59 @@
+!((define)=>{define((require,exports,module)=>{
```

Comment:
We have an option to keep it as plugin or just implement wrapper around client by ourselves if we wish. Also current plugin name is not the best :)